js/okx.js

```diff
@@ -4743,46 +4743,40 @@ module.exports = class okx extends Exchange {
         // WARNING: THIS WILL INCREASE LIQUIDATION PRICE FOR OPEN ISOLATED LONG POSITIONS
         // AND DECREASE LIQUIDATION PRICE FOR OPEN ISOLATED SHORT POSITIONS
         if ((leverage < 1) || (leverage > 125)) {
-            throw new BadRequest (this.id + ' setLeverage() leverage should be between 1 and 125');
+            throw new NotSupported (this.id + ' setLeverage () leverage should be between 1 and 125');
```

Comment:
NotSupported is usually returned when the entire method is not supported. If an arg value is beyond limits, then we should return BadRequest or something similar. I suggest reverting this line.

Reply:
@kroitor actually I have been advising to use `NotSupported` in cases where the exchange supports only some market types or a specific margin mode for that particular method, do you consider it as a wrong usage?

Reply:
> @kroitor actually I have been advising to use `NotSupported` in cases where the exchange supports only some market types or a specific margin mode for that particular method, do you consider it as a wrong usage?

I think BadRequest makes more sense